challenges/challenge-2/challenge-2.css

```diff
@@ -1,98 +1,98 @@
 /*HTML Elements*/
 body {
-  position: relative;
```

Comment:
@SyntappZ 
Are these formatting changes just from using different formatters in our editors?  @Trevis42 You did find some spaces after ':' that look great.  I'm just wondering if using a different formatter would keep things more unified.  I'm using Prettier extension on VS Code because my editor was changing all the double quotes to single quotes.  SyntappZ suggested that for me.

Reply:
I think would rather not change the indentation of this file because that would make it inconsistent with all other CSS files. Tabs are not always the same, but let's keep it at two spaces.

Reply:
Sorry about that. I use prettier, might be set up diffenent... Had a similar issue when my team was using different formatters than what I had. Sorry for the format problem.